spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -3,6 +3,8 @@
 describe Idv::SessionsController do
   render_views
 
+  let(:provider) { 'https://rp1.serviceprovider.com/auth/saml/metadata' }
+  let(:authn_context) { Saml::Idp::Constants::LOA2_AUTHNCONTEXT_CLASSREF }
```

Comment:
I know we will eventually move away from LOA names but while we are still using them my understanding is that we are supporting only LOA1 and LOA3.  Am I wrong?